src/test/java/org/jabref/logic/citationstyle/CitationStyleCacheTest.java

```diff
@@ -0,0 +1,34 @@
+package org.jabref.logic.citationstyle;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import org.jabref.model.database.BibDatabase;
+import org.jabref.model.database.BibDatabaseContext;
+import org.jabref.model.entry.BibEntry;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+
+class CitationStyleCacheTest {
+
+  private BibEntry bibEntry;
+  private List<BibEntry> entries;
+  private BibDatabase database;
+  private BibDatabaseContext databaseContext;
+  private CitationStyleCache csCache;
+
+  @Test
+  void getCitationForTest() {
+    BibEntry bibEntry = new BibEntry();
+    bibEntry.setCitationKey("test");
+    List<BibEntry> entries = new ArrayList<>();
+    entries.add(0, bibEntry);
+    BibDatabase database = new BibDatabase(entries);
```

Comment:
Can't one do the following?

```suggestion
    BibDatabase database = new BibDatabase(List.of(new BibEntry("test"));
```